terraform/modules/elasticache/redis.tf

```diff
@@ -0,0 +1,18 @@
+#tfsec:ignore:aws-elasticache-enable-backup-retention:TODO CLDC-2679 setup a snapshot retention limit
+resource "aws_elasticache_cluster" "main" {
+  #checkov:skip=CKV_AWS_134:TODO CLDC-2679 setup a snapshot retention limit
+  #TODO CLDC-2679 setup redis replicas with multi-az and automatic failover
+  #TODO CLDC-2682 setup redis logging

```

Comment:
You might have already seen, but I've created tickets to setup Redis snapshots / availability, and Redis logging later on